firestore/tests/system.py

```diff
@@ -211,11 +176,10 @@ def test_document_integer_field(client, cleanup):
             '7g': '8h',
             'cd': '0j'}
     }
-    option1 = client.write_option(exists=False)
-    document.set(data1, option=option1)
+    document.create(data1)
 
     data2 = {'1a.ab': '4d', '6f.7g': '9h'}
-    option2 = client.write_option(create_if_missing=True)
+    option2 = client.write_option(exists=True)
```

Comment:
Passing 'exists=true' to create should be disallowed. Can you add a TODO somewhere?

Reply:
This is an update.  I just checked that `document.create` cannot take options.